src/api/app/controllers/application_controller.rb

```diff
@@ -40,13 +40,13 @@ def authenticator
   end
 
   def pundit_user
-    return User.current unless User.current.is_nobody?
+    User.session! if User.session
```

Comment:
actually you want (or at least those two are equivalent):

```ruby
User.session
```

Reply:
yes, you're right. this is a victim of several rounds of patching around with the API :)